taxi/urls.py

```diff
@@ -12,7 +12,8 @@
     ManufacturerListView,
     ManufacturerCreateView,
     ManufacturerUpdateView,
-    ManufacturerDeleteView,
+    ManufacturerDeleteView, DriverCreateView, DriverDeleteView,
```

Comment:
Code Style: It's better to keep the imports neat and consistent. Each imported item should be on a new line.